app/models/etl/items.rb

```diff
@@ -1,95 +1,34 @@
-require 'gds_api/rummager'
-
 class ETL::Items
   def self.process(*args)
     new(*args).process
   end
 
-  attr_reader :new_items
+  attr_reader :content_items_service
 
   def process
-    ActiveRecord::Base.transaction do
-      raw_data = extract
-      items = transform(raw_data)
-      load(items)
-    end
+    raw_data = extract
+    items = transform(raw_data)
+    load(items)
   end
 
 private
 
   def extract
-    rummager.search_enum(
-      {
-        fields: 'content_id,title,link,description,organisations',
-      },
-      page_size: 1000,
-      additional_headers: {},
-    )
+    fields = %w[content_id base_path title description]
+    Content::ItemsService.new.fetch_all_with_default_locale_only(fields)
```

Comment:
This will also import non-english content items which is not the desired behaviour.

Reply:
This is a good spot Andrew; we will address this in a separate PR.